source/_components/utility_meter.markdown

```diff
@@ -20,6 +20,7 @@ From a user perspective, utility meters operate in cycles (usually monthly) for
 
 Some utility providers have different tariffs according to time/resource availability/etc. The utility meter enables you to define the various tariffs supported by your utility provider and accounts your consumptions in accordance. When tariffs are defined a new entity will show up indicating the current tariff. In order to change the tariff, the user must call a service, usually through an automation that can be based in time or other external source (eg. a REST sensor).
 
+Sensors created with this component are persistent, so values are retained across restarts of home assistant. The first cycle for each sensor, will be incomplete; a sensor tracking daily usage, will start to be accurate the next day after the component was activated. A sensor tracking monthly usage, will present accurate data starting the first of the next month after being added to home assistant. 
```

Comment:
Pherhaps this in a note?

Reply:
Done (although it does not show in preview as such?) Can you please check, @klaasnicolaas ?

Reply:
https://deploy-preview-8439--home-assistant-docs.netlify.com/components/utility_meter/

Reply:
thanks! Another n00b question: how would I have known that? It is different then what's shown when I hit the "preview" tab in the editor.. anyway, looks good, thanks

Reply:
You must look at the two checks below `deploy/netlify`, and click on the Details button. Then you get the Home Assistant (home) website but as new compiled version with your changes.

Reply:
thanks!